ui/src/app/views/workspace-edit/component.ts

```diff
@@ -21,48 +21,54 @@ export enum WorkspaceEditMode { Create = 1, Edit = 2, Clone = 3 }
   diseaseFocusedResearch: {
     shortDescription: 'Disease focused research',
-    longDescription: 'The primary purpose of the research is to learn more about a \
-    particular disease or disorder (for example, type 2 diabetes), a trait \
-    (for example, blood pressure), or a set of related conditions\
-    (for example, autoimmune diseases, psychiatric disorders).'
+    longDescription: 'The primary purpose of the research is to learn more about a particular \
+    disease or disorder (for example, type 2 diabetes), a trait (for example, blood pressure), \
+    or a set of related conditions (for example, autoimmune diseases, psychiatric disorders).'
   },
   aggregateAnalysis: {
     shortDescription: 'Aggregate analysis to understand variation in general population',
-    longDescription: 'The primary purpose of the research is to understand variation in the\
+    longDescription: 'The primary purpose of the research is to understand variation in the \
     general population (for example, genetic substructure of a population).'
   },
   controlSet: {
     shortDescription: 'Control Set',
-    longDescription: 'All of Us data will be used  increase the number of controls available for a\
-    comparison group (for example, a case-control study) to another dataset.'
+    longDescription: 'All of Us data will be used to increase the number of controls \
+    available for a comparison group (for example, a case-control study) to another \
+    dataset.'
   },
   ancestry: {
     shortDescription: 'Population origins or ancestry',
-    longDescription: 'The primary purpose of the research is to study the ancestry or origins\
-    of a specific population.'
+    longDescription: 'The primary purpose of the research is to study the ancestry \
+    or origins of a specific population.'
   },
   population: {
     shortDescription: 'Restricted to a specific population',
-    longDescription: 'This research will focus on a specific population group. For example: a\
-    specific gender, age group or ethnic group.'
+    longDescription: 'This research will focus on a specific population group. \
+    For example: a specific gender, age group or ethnic group.'
   },
   commercialPurpose: {
     shortDescription: 'Commercial Purpose/entity',
-    longDescription: 'The study is conducted by a for-profit entity and/or in\
+    longDescription: 'The study is conducted by a for-profit entity and/or in \
     support of a commercial activity.'
   },
   containsUnderservedPopulation: {
-    shortDescription: 'Includes an underserved population',
+    shortDescription: 'Focus on an underserved population',
     longDescription: 'This research will focus on, or include findings on, distinguishing \
     characteristics related to one or more underserved populations'
+  },
+  requestReview: {
+    shortDescription: 'Request a review of your research purpose',
+    longDescription: 'I am concerned about potential stigmatization of research participants.\
```

Comment:
`stigmatization` is marked as underlined in the doc, but not here.

Reply:
I believe it will eventually be a link, but isn't yet. I didn't make a new ticket for that one, let me do that.

Reply:
Thanks - the doc just specifies the underline as a formatting detail - having a story for the link will be helpful.